CHANGELOG.md

```diff
@@ -1,3 +1,42 @@
+## Fleet 4.51.0 (Jun 07, 2024)
+
+### Endpoint Operations
+- Added support for environment variables in configuration profiles for GitOps.
+- `fleetctl gitops --dry-run` now errors on duplicate (or conflicting) global/team enroll secrets.
+- Added `activities_webhook` configuration option to allow for a webhook to be called when an activity is recorded. This can be used to send activity data to external services. If the webhook response is a 429 error code, the webhook retries for up to 30 minutes.
+- Added Tuxedo OS to the Linux distribution platform list.
+
+### Device Management (MDM)
+- Added MDM support for iPhone/iPad.
+- Added new endpoints to configure ABM keypairs and tokens.
+- Added `GET /fleet/mdm/apple/request_csr` endpoint, which returns the signed APNS CSR needed to activate Apple MDM.
+- Added query parameter `self_service` to filter the list of software titles and the list of a host's software so that only those available to install via self-service are returned.
+- Added the device-authenticated endpoint `POST /device/{token}/software/install/{software_title_id}` to self-install software.
+- Adds a new Fleet server config variable, `FLEET_SERVER_PRIVATE_KEY`. This variable contains the private key used to encrypt the MDM certificates and keys stored in Fleet.
+- Adds the ability to automatically log off and lock out `Administrator` users on Windows hosts.
+- Adds clearer error messages when attempting to set up Apple MDM without a server private key
+  configured.
+- Updated UI to support new workflows for macOS MDM setup and credentials.
+- Updated UI to support software self-service features.
+- Updated UI controls page language and hid CTA button for users without access to turn on MDM.
+- Added UI for the global and host activities for self-service software installation.
```

Comment:
Don't know if we want to consolidate MDM Configuration Keys and Self-Service into 2 concise bullets because a lot of these are small details of those two features but sound / look disjointed when all thrown up here (and mixed) maybe even first fix would just be to sort all ABM/APNS lines together and then all the self-service lines together